repos.yaml

```diff
@@ -1047,3 +1047,8 @@ repositories:
     maintainers:
       - name: TouK Helm Charts Team
       - email: helm-charts@touk.pl
+  - name: wallyqs
```

Comment:
The name here should be the same as the name in the other file. I think they should both be `nats`. Sorry for the confusion. Can you update this?

Reply:
Thanks for checking, saw the issue. Just updated 👍